api-documentation.md

```diff
@@ -21,13 +21,13 @@ The AB2D API uses [Fast Healthcare Interoperability (FHIR)](https://www.hl7.org/
   <li class="usa-process-list__item">
     <p class="usa-process-list__heading">Use the sandbox</p>
     <p>
-      Anyone can try out AB2D in the <a href="{{ '/access-test-claims-data' | relative_url }}">sandbox</a>, a test environment that allows you to practice using the API and export sample data. 
+      The sandbox is a test environment that allows anyone to try the API and download sample claims data. Learn how to <a href="{{ '/get-a-bearer-token' | relative_url }}">get a bearer token</a> and <a href="{{ '/access-test-claims-data' | relative_url }}">access sandbox data</a>. 
```

Comment:
the 123 steps are starting to look a little wordy. What do you think about taking out the following: 
- In step 2: remove "and download sample claims data"? (and if we want to keep it, can we call it test claims data? since sample data means something else?)
- In step 3: remove "Attestation is the first step and it only takes a few minutes to complete."
![Screenshot 2024-12-18 at 3 02 35 PM](https://github.com/user-attachments/assets/82ef7993-1d03-4b97-bb2a-6cf4d4b4bdb9)

Reply:
Agreed for step 3. For step 2, how about "The sandbox environment allows anyone to try the API and download test claims data. Learn how to get a bearer token and access the sandbox. "